app/configurations/config.hb.js

```diff
@@ -315,8 +315,48 @@ export default configMerger(walttiConfig, {
           en: 'Charging stations',
           de: 'Ladestationen',
         },
-        url: '/assets/geojson/hb-layers/charging.geojson',
+        url: GEOJSON_LAYERS_BASE_URL + 'charging.geojson',
+      },
+      // bike charging stations in Stuttgart
+      {
+        name: {
+          fi: '',
+          en: 'Bicycle charging stations',
+          de: 'Fahrradladestationen',
+        },
+        url: GEOJSON_LAYERS_BASE_URL + 'bicyclechargingstation.geojson',
```

Comment:
Please do not reinsert single feature layers, which were merged in a thematic layer before